features/items.md

```diff
@@ -0,0 +1,237 @@
+
+## Item Types
+The item type defines which kind of values can be stored in that item and which commands can be send to it. Each item type has been optimized for certain components in your smart home. This optimization is reflected in the data types and commands types. Little example for better understanding: Have a look at the following table and find the itemtype "Color". A Philips Hue RGB light bulb provides you three information. Is the bulb on or off, its current brightness and the color. If you want to change one of these values you can use four commands. Switching the bulb on or off, increasing or decreasing the brightness, setting the brightness directly to a specific value and you can change the bulb's color.
+
+OpenHAB2 provides you the following item types (alphabetical order):
+
+Item Type | Description | Accepted Command Types
+----------|-------------|-----------------------
+Color     | Can be used for color information (RGB)                                  | OnOff, IncreaseDecrease, Percent, HSB
+Contact   | Can be used for sensors that return an "open" or "close" as a state. This is useful for doors, windows, etc. | OpenClose
+DateTime  | Stores a timestamp including a valid time zone.                          | DateTime
+Dimmer    | Accepts percent values to set the dimmed state. Can also be used as a switch by accepting ON/OFF commands (though this only mimics a Switch by sending 0% and 100% for ON/OFF. See 'Dimmers vs Switches' note below                                                 | OnOff, IncreaseDecrease, Percent
+Group     | Item to nest other items / collect them in groups                        | The accepted command types of a group item is the same as of the underlying base item. If none is defined, the intersection of all sets of accepted command types of all group members is used instead.
+Number    | Stores a decimal value and is usually used for all kinds of sensors, i.e. temperature, brightness, wind, etc. It can also be used as a counter or as any other thing that can be expressed as a number.                                                               | Decimal
+Player    | Allows the conrtol players, i.e. Sonos or Squeezebox.                    | PlayPause, NextPrevious, RewindFastforward
+Rollershutter | Allows the control of roller shutters, i.e. moving them up, down, stopping or setting it to close to a certain percentage. | UpDown, StopMove, Percent
+String    | Stores any kind of string or textual representation of a DateTime.       | String
+Switch    | Represents a switch that can be ON or OFF e.g. lights, etc. but can also represent virtual switches e.g. presence detection, etc. | OnOff
+
+### Dimmers vs Switches
+While a Dimmer item can accept either OnOff, IncreaseDecrease, or Percent updates, Dimmer items store their state as a Percent value.  See the following example:
+
+item:
+```xtend
+Dimmer  Light_FF_Office  "Dimmer [%s %%]"  {milight="bridge01;3;brightness"}
```

Comment:
Nice catch. The %s is for a string so wouldn't work is the output is an integer.